chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -108,7 +108,7 @@ spec:
             - --pinniped-proxy-ca-cert=/etc/pinniped-proxy-tls/ca.crt
             {{- end }}
             {{- end }}
-            - --global-repos-namespace={{ include "kubeapps.globalReposNamespace" . }}
+            - --global-repos-namespace={{ include "kubeapps.helmGlobalPackagingNamespace" . }}
```

Comment:
I haven't changed the `--global-repos-namespace` argument since cobra CLI would panic if passing an unrecognized flag (and this is something that can happen when upgrading from a previous chart version). 

I'd like to get rid of this command line "global repo" instead, but not right now for ensuring a smooth transition.